Mathlib/CategoryTheory/MorphismProperty/LiftingProperty.lean

```diff
@@ -70,20 +70,89 @@ instance rlp_isMultiplicative : T.rlp.IsMultiplicative where
     have := hj _ hp
     infer_instance
 
-lemma llp_IsStableUnderCoproductsOfShape (J : Type*) :
+lemma llp_isStableUnderCoproductsOfShape (J : Type*) :
     T.llp.IsStableUnderCoproductsOfShape J := by
   apply IsStableUnderCoproductsOfShape.mk
   intro A B _ _ f hf X Y p hp
   have := fun j ↦ hf j _ hp
   infer_instance
 
-lemma rlp_IsStableUnderProductsOfShape (J : Type*) :
+lemma rlp_isStableUnderProductsOfShape (J : Type*) :
     T.rlp.IsStableUnderProductsOfShape J := by
   apply IsStableUnderProductsOfShape.mk
   intro A B _ _ f hf X Y p hp
   have := fun j ↦ hf j _ hp
   infer_instance
 
+lemma le_llp_iff_le_rlp (T' : MorphismProperty C) :
+    T ≤ T'.llp ↔ T' ≤ T.rlp :=
+  ⟨fun h _ _ _ hp _ _ _ hi ↦ h _ hi _ hp,
+    fun h _ _ _ hi _ _ _ hp ↦ h _ hp _ hi⟩
+
+lemma gc_llp_rlp :
+    GaloisConnection (OrderDual.toDual (α := MorphismProperty C) ∘ llp)
+      (rlp ∘ OrderDual.ofDual) :=
+  fun _ _ ↦ le_llp_iff_le_rlp _ _
+
+lemma le_rlp_llp : T ≤ T.rlp.llp := by
```

Comment:
I think the naming convention wants most of these lemmas named the other way around (so this one would be called `le_llp_rlp`).

Reply:
I have made the change, but this also means that all the lemmas produced by `@[simps]` have wrong names...